lib/datadog/tracing/utils.rb

```diff
@@ -43,6 +43,34 @@ def self.reset!
       end
 
       private_class_method :id_rng, :reset!
+
+      # The module handles bitwise operation for trace id
+      module TraceId
+        MAX = (1 << 128) - 1
+
+        module_function
+
+        def next_id
+          return Utils.next_id unless Datadog.configuration.tracing.trace_id_128_bit_generation_enabled
+
```

Comment:
Document the 128 format, very rough draft: "32 bit timestamp in seconds seconds in epoch; 32-bits set to zero, currently reserved; random 64-bits"